pwiz_tools/Skyline/TestFunctional/EncyclopeDiaSearchTest.cs

```diff
@@ -0,0 +1,213 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Linq;
+using pwiz.Skyline;
+using pwiz.Skyline.Alerts;
+using pwiz.Skyline.Controls;
+using pwiz.Skyline.EditUI;
+using pwiz.Skyline.FileUI;
+using pwiz.Skyline.FileUI.PeptideSearch;
+using pwiz.Skyline.Model.Prosit.Config;
+using pwiz.Skyline.Model.Prosit.Models;
+using pwiz.Skyline.Properties;
+using pwiz.SkylineTestUtil;
+using pwiz.Skyline.Util;
+using System;
+
+namespace pwiz.SkylineTestFunctional
+{
+    [TestClass]
+    public class EncyclopeDiaSearchTest : AbstractFunctionalTestEx
+    {
+        [TestMethod]
+        public void TestEncyclopeDiaSearch()
+        {
+            TestFilesZip = @"Test\EncyclopeDiaHelpersTest.zip";
+            RunFunctionalTest();
+        }
+
+        public static bool HasPrositServer()
+        {
+            return !string.IsNullOrEmpty(PrositConfig.GetPrositConfig().RootCertificate);
+        }
+
+        private bool IsRecordMode => false;
+        private int[] FinalTargetCounts = { 15, 15, 15, 60 };
+
+        protected override void DoTest()
+        {
+            if (!HasPrositServer())
+                return;
+
+            PrepareDocument("EncyclopeDiaSearchTest.sky");
+            string fastaFilepath = TestFilesDir.GetTestPath("pan_human_library_690to705.fasta");
+
+            var searchDlg = ShowDialog<EncyclopeDiaSearchDlg>(SkylineWindow.ShowEncyclopeDiaSearchDlg);
+            RunUI(() => searchDlg.ImportFastaControl.SetFastaContent(fastaFilepath));
+            //PauseTest();
+
+            // if UseOriginalURLs, delete the blib so it will have to be freshly predicted from Prosit
+            if (Program.UseOriginalURLs)
+                File.Delete(TestFilesDir.GetTestPath("pan_human_library_690to705-z3_nce33-prosit-5950B898E945AE52AD86D9CE06220EE.blib"));
+
+            RunUI(searchDlg.NextPage); // now on Prosit settings
+
+            Settings.Default.PrositIntensityModel = PrositIntensityModel.Models.First();
+            Settings.Default.PrositRetentionTimeModel = PrositRetentionTimeModel.Models.First();
+            RunUI(() =>
+            {
+                searchDlg.DefaultCharge = 3;
+                searchDlg.DefaultNCE = 33;
+                searchDlg.MinCharge = 2;
+                searchDlg.MaxCharge = 3;
+                searchDlg.MinMz = 690;
+                searchDlg.MaxMz = 705;
+                searchDlg.ImportFastaControl.MaxMissedCleavages = 2;
+                searchDlg.SetAdditionalSetting("PercolatorTrainingFDR", "0.15");
```

Comment:
This line seems to fail in foreign languages.
I'm thinking maybe it should be:
`searchDlg.SetAdditionalSetting("PercolatorTrainingFDR", 0.15.ToString(CultureInfo.CurrentCulture));`
or maybe "SetAdditionalSetting" should be changed to take an object instead of a string.